docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -2554,16 +2554,24 @@ end
 | Yes
 | No
 | 1.0
-| -
+| 2.4
 |===
 
-Checks for multiple top-level example groups.
+Checks for multiple top-level example groups or nested example groups.
+
+This cop can be configured using the `EnforcedStyle` option.
+
+Using the `EnforcedStyle: nesting`, multiple example groups for the
+same class or module should either be nested or separated into different
+test files.
 
-Multiple descriptions for the same class or module should either
-be nested or separated into different test files.
+Using the `EnforcedStyle: splitting`, nested example group for the
+same class or module should be split into separate test files.
```

Comment:
Is this correct? I believe the cop in this mode insists on unfolding the nested example groups.